ahd2fhir/mappers/ahd_to_medication.py

```diff
@@ -30,12 +31,18 @@ def get_medication_from_annotation(annotation):
 
     # Medication Code
     codes = []
-    if "Abdamed-Averbis" in str(drug["ingredient"]["source"]):
+    if Settings().ahd_version.split(".")[0] == "5":
```

Comment:
maybe we could be more explicit with the intention behind this check by wrapping the version inside a more meaningful object: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Also note that this will break if ahd version 50.0.0 and again if version 60.0.0 is released...